src/services/authService.ts

```diff
@@ -66,11 +67,17 @@ export async function apiRegisterUser(credentials: RegisterCredentials) {
         body: JSON.stringify(credentials),
       },
     )
+
+    if (!response.ok) {
+      const errorMessage = await response.text()
```

Comment:
If the status is not `OK`, the back end returns JSON
https://github.com/Sunagatov/Iced-Latte/blob/development/src/test/resources/security/model/schema/security-schema-failed.json
It corresponds to `ErrorResponse`.